### PR TITLE
Clarify method call syntax

### PR DESCRIPTION
**Pull request by lizvdk**
Eliminate double negative to clarify method call syntax.

### CODE REVIEW

**Comment by lizvdk** [comment]
attn: @ZPendragon

File: README.md [hunk 1]
```diff
@@ -631,5 +631,3 @@ Custom fonts for the project sh
 + (UIFont *)int_navigationHeaderFontWithSize:(CGFloat)size;
 + (UIFont *)int_navigationHeaderFont;
 ```
-
```
Comment (lizvdk) [inline README.md]: whoops i will undo the whitespace changes

File: README.md [hunk 2]
```diff
@@ -55,16 +55,16 @@ view.backgroundColor = [UIColor orangeColor];
...
 ```
 
-Dot-notation should **not** be used to call methods.
+Bracket notation should be used to call methods.
 
-**For example:**
+**Do this:**
 ```objc
-array.firstObject
+[array firstObject]
```
Comment (Benuuu) [inline README.md]: This example is incorrect because firstObject is a property.
A better example would be `[set anyObject]`

Comment (lizvdk) [inline README.md]: Could I just switch the examples?

Comment (Benuuu) [inline README.md]: Yep!

Comment (Benuuu) [inline README.md]: No because you're trying to show a positive case of bracket notation.

**Comment by Benuuu** [comment]
:shipit:

**Pull request merged**